ui/src/app/components/workflows/workflow/dynamic-parts/cron-quartz-part/cron-quartz-part.component.html

```diff
@@ -67,6 +59,15 @@
 <div *ngIf="isShow">
```

Comment:
I would repleace this part with: 
`<div *ngIf="isShow">
  <clr-control-container>
    <label>{{name}}</label>
    <input *ngIf="showGuard == 1" [readOnly]="true" clrControl value="Every {{ showFrequency }} {{ cronValue }} minutes"
      [size]="40" />
    <input *ngIf="showGuard == 2" [readOnly]="true" clrControl
           value="Every {{ showFrequency }} {{ cronValue }} minutes past the hour" [size]="40" />
    <input *ngIf="showGuard == 3" [readOnly]="true" clrControl value="Every {{ showFrequency }} at {{ cronValue }}:00"
      [size]="40" />
  </clr-control-container>
</div>`

It will appear the same way as the rest of the parts.

Reply:
Btw @jozefbakus if you use three backticks, the linebreaks will be preserved in the comment